addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitAccessoryUpdater.java

```diff
@@ -74,10 +74,10 @@ public void unsubscribe(GenericItem item, String key) {
     private static interface Subscription extends StateChangeListener {
 
         @Override
-        public abstract void stateChanged(Item item, State oldState, State newState);
```

Comment:
Note the every interface defines public (abstract) methods, so it is not needed and can safely be removed.